projects/aca-content/viewer/src/lib/components/viewer/viewer.component.html

```diff
@@ -7,12 +7,13 @@
     [nodeId]="nodeId"
     [versionId]="versionId"
     [allowNavigate]="navigateMultiple"
-    [allowRightSidebar]="true"
+    [allowRightSidebar]="false"
     [allowPrint]="false"
     [showRightSidebar]="true"
     [allowDownload]="false"
     [allowFullScreen]="false"
     [overlayMode]="true"
+    [closeButtonPosition]="'viewer.closeButtonPosition' | adfAppConfig: 'right'"
```

Comment:
Did you check if that works with `left` as a default value in ADF? And also if that works if the param is not set in the config file

Reply:
Yes, I checked with default value as left as well working fine and also I try after removing the param from config file, it will come at right by default.